src/server.py

```diff
@@ -562,10 +593,26 @@ def __method_called_cb(self, connection, sender, path, iface,
                 Gio.dbus_error_quark(), Gio.DBusError.UNKNOWN_METHOD,
                 "Method '%s' not available" % method)
 
+    def _resume_last_bg_player(self, uuid_):
+        if self.players[uuid_] not in self._background_players:
+            return
+        assert self.players[uuid_].type_ == "bg"
+
+        if (len(self._background_players) > 0 and
+                self._background_players[-1] == self.players[uuid_]):
+            i = len(self._background_players) - 1
+            # Resume the last player if any.
+            if len(self._background_players) > 1:
+                self._background_players[-2].play()
```

Comment:
We probably don't need this now yet, but I would pop before doing this, so that we maintain the invariant that `_background_players[-1]` is always the one playing.